src/translator/scorers.h

```diff
@@ -25,22 +25,22 @@ class Scorer {
   Scorer(const std::string& name, float weight)
       : name_(name), weight_(weight) {}
 
-  std::string getName() { return name_; }
-  float getWeight() { return weight_; }
+  std::string getName() const { return name_; }
+  float getWeight() const { return weight_; }
 
-  virtual void clear(Ptr<ExpressionGraph>) = 0;
+  virtual void clear(Ptr<ExpressionGraph>) const = 0;
```

Comment:
Also here, what does it mean to clear() the object, if this is not a static method, and it is const? What gets cleared?

Would you mind adding a comment?

Reply:
the graph, as far as I understand. Precisely the reason why I'm proposing these changes. If calling clear() doesn't change the scorer, 'clear' should be a const function.

Reply:
Would you mind adding a brief comment to the code?